style.css

```diff
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+* {
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    display: flex;
+    justify-content: space-around;
+    background-color: #f2f2f2;
+    padding: 20px 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1;
+}
+
+nav ul {
+    display: flex;
+    list-style-type: none;
+    justify-content: space-between;
+}
+
+nav {
+    width: 25%;
+}
+
+.nav-item {
+    color: #696969;
+    font-size: 13px;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', sans-serif;
+}
+
+.icons-container {
+    width: 10%;
+}
+
+.icons {
+    display: flex;
+    list-style-type: none;
+    justify-content: space-between;
+    color: gray;
+}
+
+.landing-container {
+    position: relative;
+    margin-top: 70px;
```

Comment:
Look here how there is a white space between the image and the navbar, and this doesn't look really good for the design. 
![image](https://user-images.githubusercontent.com/54964739/95333930-1982ba80-08b6-11eb-9a83-61044b542692.png)

Try to remove the `margin-top: 70px;` from your code and see how your design will looks better!